tsconfig.json

```diff
@@ -1,4 +1,9 @@
 {
+  "compilerOptions": {
+    "jsx": "react",
+    "strict": true,
+    "noImplicitAny": true,
```

Comment:
Remove the trailing comma after `"noImplicitAny": true`. JSON does not allow trailing commas, and this could lead to parsing errors.